### PR TITLE
[Improvement] Improve docstring coverage

### PR DESCRIPTION
**Pull request by Dai-Wenxun**
## Modification

Improve docstring coverage.

image

### CODE REVIEW

File: mmaction/structures/bbox/bbox_target.py [hunk 1]
```diff
@@ -1,20 +1,22 @@
...
 
     Args:
-        pos_bboxes_list (List[Tensor]): Positive bboxes list.
-        neg_bboxes_list (List[Tensor]): Negative bboxes list.
-        gt_labels (List[Tensor]): Groundtruth classification label list.
-        cfg (dict): RCNN config.
+        pos_bboxes_list (List[torch.Tensor]): Positive bboxes list.
+        neg_bboxes_list (List[torch.Tensor]): Negative bboxes list.
+        gt_labels (List[torch.Tensor]): Groundtruth classification label list.
+        cfg (Union[dict, mmengine.ConfigDict]]): RCNN config.
```
Comment (ly015) [inline mmaction/structures/bbox/bbox_target.py:19]:
```suggestion
        cfg (dict | mmengine.ConfigDict): RCNN config.
```

File: mmaction/models/utils/graph.py [hunk 2]
```diff
@@ -3,6 +3,7 @@
 
 
 def get_hop_distance(num_node, edge, max_hop=1):
+    """Get n-hop distance matrix by edges."""
```
Comment (ly015) [inline mmaction/models/utils/graph.py:6]: Please add details descriptions like arguments and return values for utility functions.

**Comment by ly015** [comment]
@Dai-Wenxun Please resolve the conflict and this pr can be merged.

File: mmaction/datasets/transforms/processing.py [hunk 3]
```diff
@@ -13,10 +13,12 @@
 
 
 def _combine_quadruple(a, b):
+    """Utility function for combining quadruple."""
```
Comment (Dai-Wenxun) [inline mmaction/datasets/transforms/processing.py:16]:
```suggestion
```

File: mmaction/datasets/transforms/processing.py [hunk 4]
```diff
@@ -13,10 +13,12 @@
...
     return a[0] + a[2] * b[0], a[1] + a[3] * b[1], a[2] * b[2], a[3] * b[3]
 
 
 def _flip_quadruple(a):
+    """Utility function for flipping quadruple."""
```
Comment (Dai-Wenxun) [inline mmaction/datasets/transforms/processing.py:21]:
```suggestion
```

File: mmaction/models/backbones/agcn.py [hunk 5]
```diff
@@ -21,11 +22,13 @@ def conv_branch_init(conv, branches):
 
 
 def conv_init(conv):
+    """Utility function for initialize conv."""
```
Comment (Dai-Wenxun) [inline mmaction/models/backbones/agcn.py:25]:
```suggestion
```

File: mmaction/models/backbones/agcn.py [hunk 6]
```diff
@@ -12,6 +12,7 @@
 
 
 def conv_branch_init(conv, branches):
+    """Utility function for initialize conv branch."""
```
Comment (Dai-Wenxun) [inline mmaction/models/backbones/agcn.py:15]:
```suggestion
```

File: mmaction/models/backbones/agcn.py [hunk 7]
```diff
@@ -21,11 +22,13 @@ def conv_branch_init(conv, branches):
 
 
 def conv_init(conv):
+    """Utility function for initialize conv."""
     kaiming_init(conv.weight)
     constant_init(conv.bias, 0)
 
 
 def bn_init(bn, scale):
+    """Utility function for initialize bn."""
```
Comment (Dai-Wenxun) [inline mmaction/models/backbones/agcn.py:31]:
```suggestion
```

**Review by ly015 — APPROVED** [review]

**Pull request merged**